external/fv3fit/fv3fit/emulation/models/transformed_model.py

```diff
@@ -74,12 +72,18 @@ def build(
         # contains both inputs and outputs the serialized model will think its
         # outputs are also inputs and never be able to evaluate...even though
         # calling `model(data)` works just fine.
+        input_names = set(
```

Comment:
can't use the `backwards_names` method any more since that was moved to `TransformFactory`, but it is still possible to infer the input names using this logic.